spec.bs

```diff
@@ -121,10 +121,10 @@ Origins may ask browser to fetch tokens from the issuers of their
 choice. Tokens can be redeemed from a different origin than the fetching one.
 
 Private State Tokens API performs cross site anonymous authentication without
-using linkable state carrying cookies [[RFC6265]]. Cookies do provide cross
+using linkable state carrying cookies [[!RFC6265]]. Cookies do provide cross
```

Comment:
```suggestion
using linkable state carrying cookies [[RFC6265]]. Cookies do provide cross
```

Reply:
I think these are purely informative, since you don't need to implement cookies to do PSTs